automation_tools/satellite6/log.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
```

Comment:
According to the PEP 263 [1] the proper format is `# coding=<encoding name>`. The other formats `# -*- coding: <encoding name> -*-` and `# vim: set fileencoding=<encoding name> :` are editor specific and supported by Python as well.

This is not a blocker but a curiosity, I've started using the Python specific format on my files and wanted to share this info.

[1] https://www.python.org/dev/peps/pep-0263/